jobs/base_model_validator_test.py

```diff
@@ -41,6 +41,10 @@ class MockModel(base_models.BaseModel):
     pass
 
 
+class MockCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
```

Comment:
Please use the real model instead, and delete the mock.

Reply:
Done